package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "setup-beam",
```

Comment:
Not really sure where this stuff comes from. 🤷‍♂️

Reply:
wdym?

Reply:
@starbelly, is this Ok to merge, still? Thanks much.

Reply:
> Not really sure where this stuff comes from. 🤷‍♂️

It comes from Node.js 14 with `npm install`.